mlfinlab/portfolio_optimization/risk_estimators.py

```diff
@@ -258,3 +260,197 @@ def denoise_covariance(self, cov, tn_relation, kde_bwidth):
         cov_denoised = self.corr_to_cov(corr, np.diag(cov) ** (1 / 2))
 
         return cov_denoised
+
+    @staticmethod
+    def minimum_covariance_determinant(returns, price_data=False, assume_centered=False,
+                                       support_fraction=None, random_state=None):
+        """
+        Calculates the Minimum Covariance Determinant for a dataframe of asset prices or returns.
+
+        This function is a wrap of the sklearn's MinCovDet (MCD) class.
+
+        :param returns: (pd.dataframe) Dataframe where each column is a series of returns or prices for an asset.
```

Comment:
Under what conditions would you build a cov mat on prices? Are these prices then transformed to returns or it uses raw prices?

Reply:
It looks like it transforms price to rets. You should specify in the doc string that it applies that transformation.

Reply:
Will do